Core/SaveState.cpp

```diff
@@ -412,18 +412,19 @@ namespace SaveState
 		return filename.GetFilename() + " " + sy->T("(broken)");
 	}
 
-	Path GenerateSaveSlotFilename(const Path &gameFilename, int slot, const char *extension)
-	{
+	std::string GenerateFullDiskId(const Path &gameFilename) {
```

Comment:
nit: This is called disc everywhere else.  Personally, I usually use "disk" for storage, and disc for CDs and UMDs.  But either way, I think we should be consistent here.

-[Unknown]